plans/destroy.pp

```diff
@@ -14,20 +14,21 @@
 #   The name of the profile to be used for provisioning
 #
 plan provision::destroy(
-  Provision::CloudProvider  $provider      = 'aws',
-  String[1]                 $resource_name = 'puppetlabs-provision',
-  String[1]                 $profile       = undef,
-  Optional[String[1]]       $region        = 'us-west-2',
+  Provision::CloudProvider  $provider          = 'aws',
+  String[1]                 $resource_name     = 'puppetlabs-provision',
```

Comment:
Do we care that the resource name has a different default value here than in the create plan?

Reply:
good catch, looks I missed to update in destroy. Thanks for pointing.